test/verifiedTests/testTopology/testMoieties.m

```diff
@@ -54,6 +62,8 @@
     % Estimate chemical formulas of decomposed moieties
     [decomposedMoietyFormulas, M] = estimateMoietyFormulas(D, E, elements);
     assert(all(strcmp(decomposedMoietyFormulas, decomposedMoietyFormulas0)), 'Estimated formulas of decomposed moieties do not match reference.')
+
```

Comment:
We will have to remove as well the path after the test using `rmpath(genpath(path_GUROBI)` ...